homeassistant/components/blink/alarm_control_panel.py

```diff
@@ -61,7 +59,7 @@ def supported_features(self) -> int:
     @property
     def name(self):
         """Return the name of the panel."""
-        return f"{BLINK_DATA} {self._name}"
+        return f"{DOMAIN} {self._name}"
```

Comment:
I don't think that we need to use DOMAIN and `self._name` would be enough?

Reply:
The UI makes it pretty clear to which integration an entity belongs. From more info you can go to settings -> related. You can also see it in the devices and entities list. I think that it will end up with more people having to rename their entities.